src/tests/end-to-end/common/page-controllers/page.ts

```diff
@@ -138,6 +138,15 @@ export class Page {
         });
     }
 
+    public async waitForAndClickSelector(selector: string): Promise<void> {
+        await this.waitForSelector(selector);
+        await this.clickSelector(selector);
+    }
+
+    public deleteFile(filePath: string): void {
```

Comment:
This can be dropped if you accept the other comment suggesting removing the `saveAs` step

Reply:
per our conversation this has been left in.